.github/workflows/main.yml

```diff
@@ -153,6 +154,7 @@ jobs:
       shell: bash
       run: |
         datacube system check
+        datacube metadata add "https://raw.githubusercontent.com/GeoscienceAustralia/dea-config/master/product_metadata/eo3_sentinel_ard.odc-type.yaml"
```

Comment:
Any there any other metadata types added elsewhere? If not, why is this one different? If there are, should this belong there?

Reply:
We require `metadata type` to add ls and s2 C3 products (of DEA). I don't find a script to gather the database init things. It's a quick fix to make sure everything is working.